MAESTRO/Snakemake/scATAC/Snakefile

```diff
@@ -200,10 +154,21 @@ if config["platform"] == "microfluidic":
             "Result/Benchmark/%s_QCMerge.benchmark" %(config["outprefix"])
         shell:
             "python " + SCRIPT_PATH + "/scATAC_microfluidic_QC.py --log-dir {params.log} --directory {params.outdir};"
+    
+    rule scatac_celldedup:
+        input:
+            frag_count = "Result/minimap2/fragments_corrected_count.tsv",
+        output:
+            frag_dedup = "Result/minimap2/fragments_corrected_celllevel_dedup.bed",
+        benchmark:
+            "Result/Benchmark/%s_Celldedup.benchmark" %(config["outprefix"])
+        shell:
+            "cut -f 1,2,3 {input.frag_count} > {output.frag_dedup};"
 
     rule scatac_allpeakcall:
         input:
-            bam = "Result/minimap2/%s.sortedByPos.rmdp.CBadded.bam" %(config["outprefix"])
+            frag_dedup = "Result/minimap2/fragments_corrected_celllevel_dedup.bed",
```

Comment:
peaks are called using the deduplicated fragment file. in peak count, we should also use the deduplicated fragment file rather than using the original fragment file as in https://github.com/liulab-dfci/MAESTRO/blob/master/MAESTRO/Snakemake/scATAC/Snakefile#L872